WhiteDentalClinic/WhiteDentalClinic.Application/Services/AppointmentService.cs

```diff
@@ -15,54 +15,66 @@ public AppointmentService(IAppointmentRepository appointmentRepository, IMapper
             _appointmentRepository= appointmentRepository;
             _mapper= mapper;
         }
-        public IEnumerable<AppointmentResponseModel> GetAllAppointments()
+
```

Comment:
Way too many useless empty lines added in this file. some cleanup would be nice.